content/languages/cpp/igloo/stringizers.md

```diff
@@ -65,25 +65,200 @@ Test output:
 ```
   does_not_pretty_print_vector_of_type_without_stringizer
     Expected: equal to [ [unsupported type], [unsupported type] ]
-    Actual: [ [unsupported type] ]
+    Actual: [ [unsupported type], [unsupported type] ]
 ```
 
 Not only your custom types can be affected by this issue. Many built-in, standard, or 3rd party types also cannot be stringified. Basically, every type which does not define its version of output stream operator (`operator <<`) is affected, and `std::pair` template is a very common case of such type for Codewars kata.
```

Comment:
If they "cannot be stringified", we wouldn't be able to stringify them either.

Reply:
```suggestion
Not only your custom types can be affected by this issue. Snowhouse may be not able to stringify many built-in, standard, or 3rd party types. Basically, every type which does not define its version of output stream operator (`operator <<`) is affected, and `std::pair` template is a very common case of such type for Codewars kata.
```